README.md

```diff
@@ -194,3 +194,19 @@ All the results are written into JSON files that are found in a `results` folder
 ```
 
 Every JSON file contain information about the test results (status, output, memory usage), environments (used hashes, commit messages) and the main section sizes of the application (iotjs or jerryscript) binary.
+
+### Run iotjs tests with coverage measurement on Artik053
+
+You can able to run coverage measurement on Artik053. It uses the jerry-debugger to calculate the covered JS source lines, so needs to specify a unique network address for the connection with the `coverage` option. Artik053 use wifi for the communication, so also needs to set the following enviroment variables:
```

Comment:
You are able to run ... on ARTIK 053.
A modified jerry-debugger is used to calculate ... so it needs to specify ...
ARTIK 053 uses wifi ... so it also needs ...